fixtures/monorepo/host/BundleRegistry.js

```diff
@@ -0,0 +1,72 @@
+import EventEmitter from 'react-native/Libraries/vendor/emitter/EventEmitter';
+import BatchedBridge from 'react-native/Libraries/BatchedBridge/BatchedBridge';
+
+class BundleRegistry extends EventEmitter {
```

Comment:
Is it possible to offload this logic to Haul?

Reply:
It is, but I don't want to do it right now, but later, once we have everything in RN that we need to creating native module for multi-bundle support